observer.py

```diff
@@ -53,9 +52,12 @@ async def main(args):
 
     publishers = get_publishers(args.network)
     coingecko_prices = {}
-    gprice = Gauge(
+    prom_prices = Gauge(
         "crypto_price", "Price", labelnames=["symbol", "publisher", "status"]
     )
+    prom_publisher_status = Gauge(
+        "publisher_status", "Publisher Status", labelnames=["symbol", "publisher", "error_code"]
```

Comment:
maybe call this "publisher_errors"? We already have a "status" per publisher, so this could be confusing.

Reply:
Good idea.